svc.go

```diff
@@ -19,6 +19,8 @@ import (
 	manet "github.com/multiformats/go-multiaddr/net"
 )
 
+var streamReadTimeout = 60 * time.Second
```

Comment:
can we get timeout in one place, so it can be set as a config option by a consumer consistently, rather than being defined in each sub module as a variable that can't be changed?

Reply:
@willscott That change will need changes across submodules now. Please can we merge this and this later ? Have created a follow-up issue at https://github.com/libp2p/go-libp2p/issues/1035 and assigned it to myself.

Reply:
yup. just wanted to raise that the downside is now it's basically impossible to change this